eodhp_utils/pulsar/client.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+
+from pulsar import Client as PulsarClient
+
+
+class Client:
+    def __init__(self, url=None):
+        if url is None:
+            url = os.environ.get("PULSAR_URL")
```

Comment:
don't set an env var here, the code creating the client can get it from env but the client should always expect it to be given as an input to the __init__ function.